climada/entity/exposures/base.py

```diff
@@ -700,6 +709,11 @@ def write_raster(self, file_name, value_name='value', scheduler=None):
             raster, meta = u_coord.points_to_raster(self, [value_name], scheduler=scheduler)
             u_coord.write_raster(file_name, raster, meta)
 
+    def _replace_if_by_impf(self):
+        LOGGER.info('Replacing if_ by impf_ in Exposure geodataframe')
+        column_name = self.gdf.columns[self.gdf.columns.str.startswith('if_')][0]
+        self.gdf.rename(columns={column_name: column_name.replace("if", "impf")}, inplace=True)
```

Comment:
Be careful, it is valid for an exposure to have more than one column `if_`. For example, you could have `if_TC` and `if_WF` for Tropical Cyclones and Wildfires. This will become increasingly important for multi-hazard analysis. The above code would fail in this case and only update the first instance. 

I suppose this might be a problem in other parts of the code too.